documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -309,6 +309,67 @@ Should you want to make a property sortable but not searchable, still an `@Field
 bridge configuration can be inherited). It can be marked with `store = Store.NO` and `index = Index.NO`, causing
 only the doc value field required for sorting to be added, but not a regular index field.
 
+Fields added through class-level bridges or custom field-level bridges (when not using the default field name) cannot
+be marked as sortable by means of the `@SortableField` annotation. Instead the field bridge itself has to add the
+required doc value fields, in addition to the document fields it adds. Furthermore such bridge needs to implement the
+`MetadataProvidingFieldBridge` interface which defines a method `getSortableFieldNames()` for returning the names of
+the sortable fields created by this bridge:
+
+[[example-adding-docvaluefield]]
+.Adding a sortable field via a custom field bridge
+====
+[source, JAVA]
+----
+public class MyClassBridge implements MetadataProvidingFieldBridge {
+
+    @Override
+    public void set(
+            String name, Object value, Document document, LuceneOptions luceneOptions) {
+
+        // add other document field(s)
+        // ...
+
+        // add doc value field
+        document.add( new NumericDocValuesField( "myField", 311 );
+    }
+
+    Set<String> getSortableFieldNames(String name) {
```

Comment:
Ah, I think I got it. The fields returned are the one created for a particular field. isn't it?